### ISSUE TITLE
pg_pathman breaks pg_repack

### ISSUE THREAD

**Issue by tarvip** [postgrespro/pg_pathman#118]
Steps to reproduce:
```
postgres@ip-10-1-1-68:~$ createdb foo
postgres@ip-10-1-1-68:~$ psql foo -c "create table t(a bigint primary key, b text);"
CREATE TABLE
postgres@ip-10-1-1-68:~$ psql foo -c "create extension pg_repack"
CREATE EXTENSION
postgres@ip-10-1-1-68:~$ pg_repack -t t -d foo
INFO: repacking table "t"
postgres@ip-10-1-1-68:~$ psql foo -c "create extension pg_pathman"
CREATE EXTENSION
postgres@ip-10-1-1-68:~$ pg_repack -t t -d foo
INFO: repacking table "t"
ERROR: query failed: ERROR:  SET TRANSACTION ISOLATION LEVEL must be called before any query
DETAIL: query was: BEGIN ISOLATION LEVEL SERIALIZABLE
postgres@ip-10-1-1-68:~$ psql foo -c "drop extension pg_pathman"
DROP EXTENSION
postgres@ip-10-1-1-68:~$ pg_repack -t t -d foo
INFO: repacking table "t"
postgres@ip-10-1-1-68:~$
```
postgres version 9.5.8
pg_pathman 1.4.3
pg_repack 1.4.0.1

**Comment by funbringer** [comment]
Hi @tarvip,

Thanks for the report, we'll solve this issue ASAP.

**Comment by funbringer** [comment]
I've managed to fix this (see `master_hotfix_issue_118` branch):

```bash
./bin/pg_repack -dissue_118 -tt
INFO: repacking table "t"

./bin/pg_repack -dissue_118 -It
INFO: repacking table "t"
INFO: repacking table "t_1"
INFO: repacking table "t_2"
INFO: repacking table "t_3"
INFO: repacking table "t_4"
INFO: repacking table "t_5"
INFO: repacking table "t_6"
INFO: repacking table "t_7"
INFO: repacking table "t_8"
INFO: repacking table "t_9"
INFO: repacking table "t_10"
```

Perhaps we should add some tests for pg_repack to our CI.

**Comment by tarvip** [comment]
Yup, now it is okay and thank you for quick fix.

**Comment by funbringer** [comment]
Released [1.4.4](https://github.com/postgrespro/pg_pathman/releases/tag/1.4.4).
